src/passes/remove-namespace-initializers.ts

```diff
@@ -14,8 +14,36 @@
 
 import * as astTypes from 'ast-types';
 import * as estree from 'estree';
-import {getMemberOrIdentifierName} from '../util';
+import {getMemberOrIdentifierName} from '../document-util';
 
+/**
+ * Detect certain types of expressions that we deep noops when looking for
```

Comment:
might be nice to have an example or two

Reply:
they're described in the code, but added one to the description for editors